rrule/examples/timezone_support.rs

```diff
@@ -4,20 +4,23 @@
 //! in the Europe/Berlin timezone, and one EXDATE that is specified
 //! in UTC and collides with one of those recurrences.
 
-use chrono::{DateTime, Local, TimeZone};
-use chrono_tz::{Europe::Berlin, Tz, UTC};
-use rrule::{Frequency, RRule};
+use chrono::{DateTime, TimeZone};
+use rrule::{Frequency, RRule, Tz};
 
 fn main() {
+    let tz: Tz = chrono_tz::Tz::Europe__Berlin.into();
```

Comment:
I think it makes sense to re-export `chrono_tz::Tz::*`, in `rrule::Tz`.
I'm not sure if it will work without any issues 🤔

Reply:
Do you mean adding every timezone in `chrono_tz::Tz` as a variant to `rrule::Tz`?

Reply:
I meant that, but I'm not sure if it's a good pattern or not.

Reply:
I am not sure if it is an anti-pattern or not. But I did it and I think it improves the usability of the `rrule::Tz` quite a bit!

Reply:
Cool 🙏